lang/en/block_qrcode.php

```diff
@@ -39,4 +39,5 @@
 $string['submit'] = 'Save changes';
 $string['editblock'] = 'Edit QR code block';
 $string['usedefault'] = 'Use default settings';
-$string['instc_uselogo'] = 'Use logo';
\ No newline at end of file
+$string['instc_uselogo'] = 'Use logo';
+$string['privacy:metadata'] = 'The qrcode block only displays information about a course, but does not effect or store any personal data. All displayed logos are managed by the file manager.';
```

Comment:
I wouldn't consider logos personal data, given that they are applied institution-wide. Therefore, the statement is correct but misleading and should therefore be removed.

By the way, if it **were** personal data, you would need to declare this explicitly within the provider, i.e. the `null_provider` would probably not be the correct choice.